modules/social_features/social_user/social_user.module

```diff
@@ -675,32 +675,38 @@ function social_user_ensure_help_text_fieldset(&$form) {
  * displays the one with the lowest weight (highest priority).
  */
 function social_user_user_format_name_alter(&$name, AccountInterface $account) {
-  // We always add the username as fallback suggestion.
-  $suggestions = [
-    'username' => [
-      'name' => $account->getAccountName(),
-      'weight' => PHP_INT_MAX,
-    ],
-  ];
+  $suggested_names = &drupal_static(__FUNCTION__, []);
+
+  if (!isset($suggested_names[$account->id()])) {
+    // We always add the username as fallback suggestion.
+    $suggestions = [
+      'username' => [
+        'name' => $account->getAccountName(),
+        'weight' => PHP_INT_MAX,
+      ],
+    ];
 
-  $suggestions += \Drupal::moduleHandler()->invokeAll('social_user_name_display_suggestions', [$account]);
-  \Drupal::moduleHandler()->alter('social_user_name_display_suggestions', $suggestions, $account);
+    $suggestions += \Drupal::moduleHandler()->invokeAll('social_user_name_display_suggestions', [$account]);
+    \Drupal::moduleHandler()->alter('social_user_name_display_suggestions', $suggestions, $account);
 
-  // We could use -PHP_INT_MAX but someone being clever might shoot themselves.
-  $lowest_weight = NULL;
+    // We could use -PHP_INT_MAX but someone being clever might shoot themselves.
```

Comment:
PHPCS complained, since the added 2 spaces now push the line over 80 characters 😬 
```suggestion
    // We could use -PHP_INT_MAX but someone being clever might shoot 
    // themselves.
```

Reply:
🥱